src/components/Account-panel/style.ts

```diff
@@ -4,3 +4,15 @@ export const stInput = {
     backgroundColor: 'background.default'
   }
 }
+
+export const stH3 = {
+  fontFamily: 'Comfortaa',
+  fontSize: '32px',
+  fontWeight: 500
+}
+
+export const stP = {
+  fontFamily: 'Open Sans',
+  fontSize: '18px',
+  fontWeight: 300
+}
```

Comment:
you don't need this styles, we already have this styles in typography variants

Reply:
done